tests/flow/async/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -356,13 +357,15 @@ function test3() {
 //
 
 function test4() {
-  async function voidoid5(): void { // error, void != Promise<void>
+  async function voidoid5(): void {
```

Comment:
Yep, it was a goal to keep lines at the end of function declarations:

```js
function foo(arg1, arg2) { // comment here
}
```

If we choose not to do that, we shouldn't need `globalPrecedingNode`. But people are probably going to complain that they can't disable a rule that enforces something about the arguments.